### ISSUE TITLE
Rewrite module not running

### ISSUE THREAD

**Issue by raguay** [clickontyler/VirtualHostX-Issues#8]
The apache rename module doesn't seem to be working or enabled. Apache is only accessing things that are in the directory and not redirecting to the index.php as per the .htaccess rules file in the directory. It might just need the rules added to the VHX 7 UI, but I can't test that now as I am locked out of the virtual machine right now.

**Comment by tylerhall** [comment]
Do you mean mod_rewrite?

**Comment by raguay** [comment]
Yea, I keep getting the name wrong because it rewrites by renaming. 😊

**Comment by tylerhall** [comment]
This is the beautiful thing about VHX 7. Since it's a virtual machine, you're free to login and make whatever changes you want. This includes turning on mod_rewrite...

First, ssh into the virtual machine

```
ssh vagrant@172.17.0.10
```

The password is "vagrant".

Then....

```
sudo a2enmod rewrite
sudo service apache2 restart
```

And that should do it.

**Comment by raguay** [comment]
I did the above and it still didn't work because I have special header directives in the .htaccess file. Then I added:

sudo a2enmod headers

and added to the Directory directive on the virtual machine:

Options Indexes FollowSymLinks MultiViews
AllowOverride All
Order allow,deny
allow from none

Note: The setting the directory directive in the UI did nothing. I had to do it on the virtual machine manually.

Now, I have the server working like VHX6 did before. I would suggest adding those two modules to the standard install and getting the directives to work from the gui. Then, VHX7 will be a great solution out of the box.

Also, you might want to create your own box and install that instead of the standard Ubuntu. I have seen the vagrant configuration for this system not work once. But destroying and reinstalling did finally work.
